### PR TITLE
Add changes to skip update option

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Update info on Skip update option

### CODE REVIEW

**Comment by thaJeztah** [comment]
the new screenshot looks to be a lot bigger than the previous one (dimensions)

File: _includes/desktop-update.md [hunk 1]
```diff
@@ -2,10 +2,8 @@
 
 Starting with Docker Desktop 3.0.0, updates to Docker Desktop will be available as delta updates from the previous version. When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. You can choose when to start the download and installation process.
 
-> To encourage developers to stay up to date, Docker Desktop displays a reminder two weeks after an update becomes available. You can dismiss this daily reminder by clicking **Snooze**.
+> To encourage developers to stay up to date, Docker Desktop displays a reminder two weeks after an update becomes available. You can dismiss this daily reminder by clicking **Snooze**. All users can now skip an update when a reminder appears by clicking the **Skip this update** option.
```
Comment (mat007) [inline _includes/desktop-update.md:5]: Do we really need to emphasize that’s it’s now all users?
Can’t we just say something like
```suggestion
> To encourage developers to stay up to date, Docker Desktop displays a reminder two weeks after an update becomes available. You can dismiss this daily reminder by clicking **Snooze**. You can skip an update when a reminder appears by clicking the **Skip this update** option.
```
?

Comment (thaJeztah) [reply _includes/desktop-update.md:5]: Yes; was thinking that as well 👆

Comment (usha-mandya) [reply _includes/desktop-update.md:5]: Yea, I was't sure, but thought it highlighted the change for existing users. Perhaps this is unnecessary background info for new users?

Comment (usha-mandya) [reply _includes/desktop-update.md]: Updated

**Comment by usha-mandya** [comment]
Optimized the image

**Review by dieucao — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**